bull/pdl-queue/producer.js

```diff
@@ -16,6 +16,7 @@ module.exports = async (bookid, categoryID, email) => {
   const metaData = await getPDLMetaData(options, bookid, categoryID);
 
   metaData["email"] = email;
+  metaData["userName"] = userName;
```

Comment:
@Parthiv-M While we're logging at the job data level, we should also ensure logging is done during `job.log()`. Please do this across all the 3 libraries. I would suggest make a generic function so that all future libraries support it.